doc/source/working_with_kiwi/shell_scripts.rst

```diff
@@ -263,6 +263,25 @@ $kiwi_type
 Configuration Tips
 ------------------
 
+#. **Locale configuration:**
+
+  KIWI in order to set the locale relies on :command:`systemd-firstboot`,
+  which in turn writes the locale configuration file :file:`/etc/locale.conf`.
+  The values for the locale settings are taken from the description XML
+  file in the `<locale>` element under `<preferences>`.
+
+  KIWI assumes systemd adoption to handle these locale settings, in case the
+  build distribution does not honor `/etc/locale.conf` this is likely to not
+  produce any effect on the locale settings. Note this is the case for old
+  distributions such as SLE12 in which the locale configuration is expected
+  to happen under the :file:`/etc/sysconfig/language` file which is,
```

Comment:
Note `/etc/locale.conf` *does* work on SLE 12. It's just that if `/etc/sysconfig/language` exists, the SUSE-specific hacks kick in.

Reply:
Yes you are right. I did not want to get into details of SLE12 behavior here but just mention it as an example of a distro that does not fully embrace the locale settings via systemd.

Reply:
Could you please tweak this to be more accurate, because I don't want to give the impression this doesn't work in SLE 12.

Reply:
Sure, what about now? I believe this is explicit enough

Reply:
LGTM.